features/support/rails.rb

```diff
@@ -1,5 +1,5 @@
 require 'cucumber/rails/application'
-require 'cucumber/rails/action_controller'
+require 'cucumber/rails/action_dispatch'
```

Comment:
If you have a moment, I was just curious why we had to change this?

Reply:
Oh, sorry, I thought I had explained this in the commit message, I added the explanation now to the commit too.

The explanation is that the file was renamed in the `cucumber-rails` gem. We didn't need some components of `cucumber-rails`, and some were causing issues for us so [I started doing what we're doing now](https://github.com/activeadmin/activeadmin/commit/ce0df9179e829faa2e7fcd2e125f07715cbd9e1f). Since that's not the "official way" of requiring `cucumber-rails`, I don't think it makes sense to complain about anything to the gem, so I catched up with the rename.

Reply:
That makes sense, thanks!